packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -90,7 +142,8 @@ contract OVM_L1CrossDomainMessenger is iOVM_L1CrossDomainMessenger, Abs_BaseCros
         override
         public
         nonReentrant
-        onlyRelayer()
+        onlyRelayer
+        whenNotPaused
```

Comment:
Note: this is the only pausable function. It's neither on `sendMessage` nor `replayMessage()`

Reply:
@karlfloersch is this right in terms of the chugsplash and deployment flow, or are we interested in pausing sending messages too?

Reply:
Thinking through this more, I believe that the pausing will occur anyway due to the nature of the chugsplash deployments.  Therefore, I think this is fine.